draft-ietf-tls-tls13.md

```diff
@@ -2729,6 +2729,10 @@ TLS also allows other messages to be sent after the main handshake.
 These messages use a handshake content type and are encrypted under the application
 traffic key.
 
+Handshake messages sent after the handshake MUST NOT be interleaved with
+application data. That is, if a message is split over two or more handshake
+records, there MUST NOT be any application data records between them.
```

Comment:
Good point. What about alerts? I could imagine wanting to allow sending:

```
[hs:NewSessio-]
[alert:internal_error]
--- connection closed ---
```

Maybe it should say non-alert records? I can't imagine any use for sticking an alert in the middle other than the server spontaneously combusting mid-send and wanting to tell you about it. Though I'm also happy just saying all records because this case is silly.